decentraland/unity-renderer
# feat: combined fps panel

## What does this PR change?

Adds a new debug panel according to #2381. 

For correct functionality, we are going to wait till the notification system is merged to merge this new debug panel. Right now, the debug panel is covering completely the chat area, which is not ideal. But once the notification area is implemented, that area will dissappear; and we are going to be able to have both spaces coexisting at the same time.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/new-debug-panel&kernel-branch=feat/memory-metrics-for-debug-panel
2. Input `/showfps` in the chat window. 
3. Check that you see the panel
4. Try to drag and drop the window
5. Check that all subpanels are closable when clicking the arrow

COMMENT: Its expected in WEBGL that the Graphics Memory value remains empty

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
